lib/smart_answer_flows/coronavirus-employee-risk-assessment/questions/where_do_you_work.erb

```diff
@@ -2,6 +2,10 @@
   Where do you work?
 <% end %>
 
+<% text_for :hint do %>
+  This list does not include all types of businesses that are currently open or allowed to open soon. If your workplace is not in the list, please select the last option 'None of these'.
```

Comment:
Should we use typographical quotes around **None of these**?

Reply:
Ah, so I think we use standard `'`characters elsewhere on GOV.UK. Is there an issue with not using them?